yarn-project/acir-simulator/src/index.test.ts

```diff
@@ -153,18 +153,24 @@ describe('ACIR simulator', () => {
       const amountToTransfer = 100n;
       const abi = ZkTokenContractAbi.functions.find(f => f.name === 'transfer') as unknown as FunctionAbi;
 
-      const tree = await StandardMerkleTree.new(db, pedersen, 'privateData', PRIVATE_DATA_TREE_HEIGHT);
+      const tree: AppendOnlyTree = await StandardTree.new<StandardTree>(
+        StandardTree,
+        db,
+        pedersen,
+        'privateData',
+        PRIVATE_DATA_TREE_HEIGHT,
+      );
```

Comment:
+1 for this.

Reply:
Good idea. Addressed in dfcc2eabbccb0cb73d781d1b221a83536719ade9 . It looks nicer.

I renamed the `fromName` function to `loadTree` because the `fromName` didn't make much sense when the function stopped being directly associated with the calss.